comm/client.go

```diff
@@ -0,0 +1,86 @@
+package comm
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"strings"
+
+	"os"
+
+	"github.com/gorilla/http"
+	//"net/http"
+
+	"github.com/phomer/scheduler/accounts"
+	"github.com/phomer/scheduler/datastore"
+	"github.com/phomer/scheduler/log"
+)
+
+func NewClient() http.Client {
+	return http.DefaultClient
+}
+
+func MakeRequest(config *accounts.ClientConfig, request *Request) string {
+
+	url := "http://127.0.0.1:8000/" // TODO: Get this from config
+	client := NewClient()
+
+	buffer := datastore.Serialize(request)
+	reader := strings.NewReader(string(buffer))
+
+	status, args, read_closer, err := client.Post(url, nil, reader)
+	if err != nil {
+		switch value := err.(type) {
+		default:
+			log.Dump("error", err)
+			log.Fatal("Get", err)
+			_ = value
+		}
+	}
+
+	defer read_closer.Close()
+
+	// TODO: Do something with this data
+	_ = status
+	_ = args
+
+	data, err := ioutil.ReadAll(read_closer)
+	if err != nil {
+		fmt.Println("Buffer read err", err)
+	} else {
+		fmt.Println("Buffer:", string(data))
+	}
+
+	fmt.Println("Command is Sent", status)
+
+	return "Sometext"
```

Comment:
I assume this is a placeholder for now? It's always a bit confusing to have a mixture between finished code, work-in-progress, and placeholders when reviewing a PR. Comments help 🙂

Reply:
Yea. It needs to be switched steam stuff until the pipe is closed or a ^C is caught. Going forward, I'll try to be nicer with the TODOs for you. Normally I wouldn't check in code this rough, but these are special circumstances.

Reply:
no worries. I understand that this is work in progress, it just affects the quality of feedback I could give you.